test/functional/test_framework/test_node.py

```diff
@@ -544,7 +544,7 @@ def add_p2p_connection(self, p2p_conn, *, wait_for_verack=True, **kwargs):
 
     def num_test_p2p_connections(self):
         """Return number of test framework p2p connections to the node."""
-        return len([peer for peer in self.getpeerinfo() if peer['subver'] == MY_SUBVERSION])
+        return len([peer for peer in self.getpeerinfo() if peer['subver'] == MY_SUBVERSION.decode("utf-8")])
```

Comment:
It might be better to leave this as-is and call `encode` where it is needed. Doing so will prevent similar issues in the future. Missing an `encode`, however, can't result in a hard-to-debug issue because it is detected immediately as a type error (either by mypy or at runtime)